dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
well, what happens is i change define.amd to false so when it comes to hammer it will skip the if statement that checks for define.amd because it's false and it will be used as if it was public (normal), so hammer will be publicly available, and the argument Hammer that is passed to the function probably i forgot to remove it because we can use hammer normally without requiring it, so amd might allow us to do something like that without giving an error and works normally but i am not sure so i will check and i will let you know, when trying to log Hammer inside of the function it will probably give us the Hammer function but like i said i am not sure and i will test it, and if there is a need for change i will change!